qutebrowser/completion/completionwidget.py

```diff
@@ -211,6 +207,12 @@ def set_model(self, model):
         Args:
             model: The model to use.
         """
+        if (config.get('completion', 'auto-open') and
+            config.get('completion', 'show')):
```

Comment:
nitpick: Please indent this by four more spaces so it doesn't get mixed up with the body of the `if` when reading.